Sources/EduIDExpansion/Controllers/Onboarding/EnterPersonalInfoViewController.swift

```diff
@@ -29,27 +64,43 @@ class EnterPersonalInfoViewController: EduIDBaseViewController, ValidatedTextFie
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         stack.animate(onlyThese: [1, 2, 3, 4, 6])
+        _ = emailField.becomeFirstResponder()
+        
+        print(stack.frame)
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        //set the height of the spacer according to the view
+        if scrollView.frame.size.height > scrollView.contentSize.height + /* a small buffer */ 50 {
```

Comment:
50 would also be a good candidate to put into a constant.

Reply:
I did this